cmd/thanos/bucket.go

```diff
@@ -46,18 +46,24 @@ var (
 )
 
 func registerBucket(m map[string]setupFunc, app *kingpin.Application, name string) {
-	cmd := app.Command(name, "Inspect metric data in an object storage bucket")
+	cmd := app.Command(name, "Bucket utility commands")
 
-	objStoreConfig := regCommonObjStoreFlags(cmd, "")
-	objStoreBackupConfig := regCommonObjStoreFlags(cmd, "-backup")
+	objStoreConfig := regCommonObjStoreFlags(cmd, "", true)
 
-	// Verify command.
-	verify := cmd.Command("verify", "Verify all blocks in the bucket against specified issues")
-	verifyRepair := verify.Flag("repair", "Attempt to repair blocks for which issues were detected").
+	registerBucketVerify(m, cmd, name, objStoreConfig)
+	registerBucketLs(m, cmd, name, objStoreConfig)
+	registerBucketInspect(m, cmd, name, objStoreConfig)
+	return
+}
+
+func registerBucketVerify(m map[string]setupFunc, root *kingpin.CmdClause, name string, objStoreConfig *pathOrContent) {
+	cmd := root.Command("verify", "Verify all blocks in the bucket against specified issues")
+	objStoreBackupConfig := regCommonObjStoreFlags(cmd, "-backup", false)
```

Comment:
why do we need to use the backup bucket? Might be worth mentioning something in the description of the `verify` command.